### ISSUE TITLE
loadDiff method

### ISSUE THREAD

**Issue by sithmel** [sithmel/obj-delta#2]
Implement a method that takes 2 object and load the diff between the 2 of them

**Comment by gwpmad** [comment]
As in the following:

```
delta()
  .loadDiff('greetings', { eng: 'hello', it: 'ciao' })
  .apply({ greetings: { eng: 'hello', es: 'hola', it: 'ciao' } });

// { es: 'hola' }
```

?

**Comment by sithmel** [comment]
no, as shown here: https://github.com/sithmel/obj-delta/blob/master/tests/delta.test.js#L91
The code already works for objects. It still doesn't work for arrays.
Arrays are more complicated as we need to take into consideration if an item has been inserted, or removed from the array
